demo/index.html

```diff
@@ -384,6 +394,18 @@
         </div>
       </div>
 
+      <div class="hx-section exampleSection">
+        <div class="example">
+          <h3>Autocompleted Tag Input</h3>
+          <div id="autocomplete-tag-input"></div>
+          <script>
+            new hx.TagInput('#autocomplete-tag-input', {
+              autocompleteData: ['Jane', 'Bob', 'Dan', 'Steve', 'Alejandro']
+            })
+          </script>
+        </div>
+      </div>
```

Comment:
I found it particularly helpful to play around with this while developing, so I'd vote for adding this for every new module/feature